kaggle_environments/agent.py

```diff
@@ -55,25 +55,28 @@ def get_last_callable(raw, fallback=None):
         raise InvalidArgument("Invalid raw Python: " + repr(e))
 
 
-def build_agent(raw, environment):
-    if raw in environment.agents:
-        return environment.agents[raw]
+def build_agent(raw, builtin_agents, environment_name):
+    """
+    Returns the agent and whether the agent is picklable.
```

Comment:
Technically speaking many of these might happen to be picklable but really what we're returning here is whether we *want* to pickle the agent not whether it's possible to pickle. Maybe I should rename this to should_parallelize or something?

Reply:
In practice this only returns true for url-based agents and static action agents.